drivers/sensor/CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ add_subdirectory_ifdef(CONFIG_SGP40		sgp40)
 add_subdirectory_ifdef(CONFIG_SHTCX		shtcx)
 add_subdirectory_ifdef(CONFIG_SHT3XD		sht3xd)
 add_subdirectory_ifdef(CONFIG_SHT4X		sht4x)
+add_subdirectory_ifdef(CONFIG_SI1133		si1133)
```

Comment:
The sensor should probably be added to `tests/drivers/build_all/sensor` so the driver is built on the buildserver.